plugin-manifest-gen.html

```diff
@@ -0,0 +1,374 @@
+            margin-bottom: 5px;
+        }
+
+        .status {
+            font-size: small;
+            padding-left: 5x;
+            text-align: right;
+        }
+
+        .success {
+            color: green;
+        }
+
+        .error {
+            color: red;
+        }
+
+        .warning {
+            color: orangered;
+        }
+
+        .hint {
+            font-size: small;
+            opacity: 80%;
+            padding: 5px;
+        }
+
+        form input[type=text] {
+            margin: 5px;
+            padding: 3px;
+            width: 100%;
+            border-width: 0px 0px 2px 0px;
+            border-color: gray;
+            font-family: 'Fira Code', monospace;
+            outline: none;
+        }
+
+        form textarea {
+            margin: 5px;
+            padding: 5px;
+            width: 100%;
+            border-width: 1px 1px 2px 1px;
+            border-color: gray;
+            font-family: 'Fira Code', monospace;
+            font-size: smaller;
+            outline: none;
+        }
+
+        form input:focus-visible {
+            border-color: #333;
+        }
+
+        form fieldset {
+            opacity: 90%;
+            margin: 10px;
+        }
+
+        form {
+            display: flex;
+            flex-direction: column;
+            align-items: stretch;
+        }
+
+        #output {
+            font-family: 'Fira Code', monospace;
+            font-size: smaller;
+            line-height: 2;
+            white-space: pre-wrap;
+        }
+
+        #copy-output-container {
+            display: flex;
+            justify-content: flex-end;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="splitter-half" id="splitter-left">
+        <form class="splitter-inner">
+
+            <div class="form-section">
+                <h3>Basics:</h3>
+
+                <div class="hint">The name of your Plug-In.</div>
+                <input name="name" id="name" type="text" placeholder="My Awesome Plug-In">
+
+                <div class="hint">One or more authors.</div>
+                <input name="authors" id="authors" type="text" placeholder="Michael Zahniser & Contributors">
+            </div>
+
+            <div class="form-section">
+                <h3>Homepage:</h3>
+
+                <div class="hint">The homepage of your Plug-In. If you don't have a dedicated homepage, it is common to
+                    use your Github Repository.</div>
+                <input name="homepage" id="homepage" type="text"
+                    placeholder="https://github.com/endless-sky/endless-sky-high-dpi/">
+
+                <div class="hint">Your repository's git URL. Leave blank if you used your Github repository as homepage,
+                    or if your homepage redirects to a git URL.</div>
+                <input name="update-url" id="update-url" type="text"
+                    placeholder="https://github.com/endless-sky/endless-sky-high-dpi.git">

```

Comment:
`<input>` without an associated `<label>` is an accessibility error.

There's a bunch of these, so please fix them all.